### PR TITLE
Feature/inline images as attachments

### PR DESCRIPTION
**Pull request by SimonBerry555**
Using the 0.9.1 version of the plugin works well - I want any inline images in the document uploaded as attachments to the page.

### CODE REVIEW

File: src/main/java/com/github/qwazer/markdown/confluence/core/service/markdown/F.java [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 /**
  *
- * @param <R>
- * @param <P>
+ * @param <R> blah
```
Comment (qwazer) [inline src/main/java/com/github/qwazer/markdown/confluence/core/service/markdown/F.java]: [minor] remove or replace meaningless word blah

File: src/main/java/com/github/qwazer/markdown/confluence/core/service/AttachmentService.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import com.github.qwazer.markdown.confluence.core.ConfluenceException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpStatusCodeException;
+
+import java.nio.file.Path;
+
+@Service
+public class AttachmentService {
```
Comment (qwazer) [inline src/main/java/com/github/qwazer/markdown/confluence/core/service/AttachmentService.java:13]: add logging

File: src/main/java/com/github/qwazer/markdown/confluence/core/service/MainService.java [hunk 3]
```diff
@@ -20,30 +25,89 @@
...
+
+            Long pageId = pageService.postWikiPageToConfluence(page, wikiText);
+
+            for (Map.Entry<String, Path> entry : imageURIToFilePathMap.entrySet()) {
+                attachmentService.postAttachmentToPage(pageId, entry.getValue());
+            }
         }
     }
 
+    // Any images in the markdown need to be published as attachments to the page.
```
Comment (qwazer) [inline src/main/java/com/github/qwazer/markdown/confluence/core/service/MainService.java:69]: this feature and how to use 
should be part of plugin doc  (README.md)

**Review by qwazer — COMMENTED** [review]
Thanks a lot for a wonderful feature. Comments has been added from my side

**Comment by qwazer** [comment]
commit message "First commit for feature." should be rephrased to provide useful info

**Comment by qwazer** [comment]
OR squash all commits together

File: src/main/java/com/github/qwazer/markdown/confluence/core/service/markdown/F.java (hunk 1 above)
Comment (SimonBerry555) [reply src/main/java/com/github/qwazer/markdown/confluence/core/service/markdown/F.java]: Done

File: src/main/java/com/github/qwazer/markdown/confluence/core/service/MainService.java (hunk 3 above)
Comment (SimonBerry555) [reply src/main/java/com/github/qwazer/markdown/confluence/core/service/MainService.java:69]: Done

File: src/main/java/com/github/qwazer/markdown/confluence/core/service/AttachmentService.java (hunk 2 above)
Comment (SimonBerry555) [reply src/main/java/com/github/qwazer/markdown/confluence/core/service/AttachmentService.java:13]: Done

**Comment by SimonBerry555** [comment]
Hi - I have addressed all the comments -would you be able to re-review and if OK merge - maybe you could squash the commits so the comment reads:
 `Add feature to process markdown inline images where the target is a file on local filesystem as attachments on the confluence page.`

Thanks

**Comment by SimonBerry555** [comment]
Hi,

I have made the changes you suggested, could you re-review and if you are
happy merge it into the codebase. When you merge you could squash the
commits and give a better comment - see my comment on the PR.

Once merged could you do a release of 0.9.2?

Any chance I could be added as a contributor as I have a couple of other
things I would like to add?

Thanks

**Pull request merged**